presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2706,6 +2706,7 @@ public List<ConnectorTableLayoutResult> getTableLayouts(ConnectorSession session
                                 false,
                                 createTableLayoutString(session, handle.getSchemaTableName(), hivePartitionResult.getBucketHandle(), hivePartitionResult.getBucketFilter(), TRUE_CONSTANT, domainPredicate),
                                 desiredColumns.map(columns -> columns.stream().map(column -> (HiveColumnHandle) column).collect(toImmutableSet())),
+                                false,
                                 false)),
```

Comment:
Can this always be false ?

Reply:
I think its safer to put it false for now